package chapter06;

public class Java03_Exception {
    public static void main(String[] args) {

        // TODO 异常
        // test

    }
}
